tests/test_stream.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python
```

Comment:
What about tests for the async io?

Reply:
I didn't add an explicit test with asyncio but I included it as a part of tests with the class TestIO, since if we a user were to give a stream from asyncio they would need to inherit in their own class in order to invoke drain after each write. That is, the flow with asyncio-stream is
```
stream.write(...)
stream.drain()
``` 
But in PyFMI we only invoke write, so they'd have to have a class inheriting their own write function, that is why I added the testclass TestIO but it instead inherits from StringIO due to simplicity.

Reply:
I see, do we have a comment that this is needed somewhere?

Reply:
We don't. We've only stated that the streams need to support write, do you think it makes sense if we do the following change?
CURRENT (docstring for load_fmu and the FMI-classes):
```
        log_file_name --
            Filename for file used to save logmessages.
            This argument can also be a stream if it supports 'write', for full functionality
            it must also support 'seek' and 'readlines'.
            The stream must also be open and writable during the entire time.
            Default: "" (Generates automatically)
```
PROPOSAL:
```
        log_file_name --
            Filename for file used to save logmessages.
            This argument can also be a stream if it supports 'write', for full functionality
            it must also support 'seek' and 'readlines'. If the stream requires use of other methods, such as 'drain' 
            for asyncio-streams, then this needs to be implemented on the user-side, there is no additional methods invoked 
            on the stream instance after 'write' has been invoked on the PyFMI side.
            The stream must also be open and writable during the entire time.
            Default: "" (Generates automatically)
```

Reply:
Looks good!

Reply:
Thanks! Added now with the latest commit.